### PR TITLE
airtable extractor documentation

### PR DESCRIPTION
**Pull request by DamienCouillard**

### CODE REVIEW

File: source/publishing_data/01_creating_a_dataset/supported_formats.rst [hunk 1]
```diff
@@ -71,6 +71,7 @@ The following table lists the supported formats. Click the link on each connecto
      *
      *
        :doc:`/publishing_data/04_configuring_a_source/connectors/xml`
+
```
Comment (NTerpo) [inline source/publishing_data/01_creating_a_dataset/supported_formats.rst:74]: You may want to remove that line

File: source/publishing_data/04_configuring_a_source/configuring_source.rst [hunk 2]
```diff
@@ -34,6 +35,9 @@ The configuration options of a source entirely depend on the source type, and bo
   * * Name
     * Description & Supported formats
     * Availability
+  * * :doc:`Airtable<connectors/airtable>`
+    * Used for data extracted from an Airtable base
+    * Default
```
Comment (NTerpo) [inline source/publishing_data/04_configuring_a_source/configuring_source.rst:40]: Shouldn't it be 'On demand' ?

File: source/publishing_data/04_configuring_a_source/connectors/airtable.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+     * By default, the textbox is extracted from the ``base_id`` field of the URL.
+   * * Table Name
+     * Name of the table to import from the database
+     * By default, the textbox is extracted from the ``table_name`` field of the URL.
+   * * API key
+     * Airtable account API key
+     * By default, the textbox is extracted from the ``api_key`` field of the URL.
+
+
+The Airtable Rest API is limited to 5 requests per second per base. Try not to scheldule updates of tables from the same base  all at once.
```
Comment (NTerpo) [inline source/publishing_data/04_configuring_a_source/connectors/airtable.rst:44]: May also be a note like above

File: source/publishing_data/04_configuring_a_source/connectors/airtable.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+     * Name of the table to import from the database
+     * By default, the textbox is extracted from the ``table_name`` field of the URL.
+   * * API key
+     * Airtable account API key
+     * By default, the textbox is extracted from the ``api_key`` field of the URL.
+
+.. admonition:: Note
+   :class: note
+   
+   The Airtable Rest API is limited to 5 requests per second per base. Try not to scheldule updates of tables from the same base  all at once.
```
Comment (vbarbaresi) [inline source/publishing_data/04_configuring_a_source/connectors/airtable.rst:46]:
```suggestion
   The Airtable Rest API is limited to 5 requests per second per base. Try not to schedule updates of tables from the same base  all at once.
```

File: source/publishing_data/04_configuring_a_source/connectors/airtable.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- your base : Choose your table then go in Help > API documentation > Authentification
+
+2. Check the ``show API key`` checkbox and copy the ``EXAMPLE USING QUERY PARAMETER`` without the curl, this is your API URL. (e.g. https://api.airtable.com/v0/base_id/table_name?api_key=api_key)
+3. Enter the URL to the layer in the Enter a URL area, when creating a new dataset.
+4. Press Enter, or click the + button.
+
+.. admonition:: Note
+   :class: note
+
+   In the example `https://api.airtable.com/v0/base_id/table_name?api_key=api_key` the ``base_id`` field stands for the ID of your Airtable database. The ``table_name`` field  stands for the name of the table to import from the database in unicode encoding.
```
Comment (vbarbaresi) [inline source/publishing_data/04_configuring_a_source/connectors/airtable.rst:22]:
```suggestion
   In the example `https://api.airtable.com/v0/base_id/table_name?api_key=api_key` the ``base_id`` field stands for the ID of your Airtable database. The ``table_name`` field stands for the name of the table to import from the database in unicode.
```
"unicode encoding" sounds redundant IMO

**Pull request merged**